src/Samhammer.AzureBlobStorage/Services/AzureBlobStorageService.cs

```diff
@@ -83,6 +98,40 @@ public async Task UploadBlobAsync(string blobName, string contentType, Stream co
             await blobClient.UploadAsync(content, options);
         }
 
+        private Uri CreateServiceSASBlob(
+            BlobClient blobClient,
+            DateTimeOffset expiresOn,
+            string storedPolicyName = null)
+        {
+            if (blobClient.CanGenerateSasUri)
+            {
+                BlobSasBuilder sasBuilder = new BlobSasBuilder()
+                {
+                    BlobContainerName = blobClient.GetParentBlobContainerClient().Name,
+                    BlobName = blobClient.Name,
+                    Resource = "b",
+                };
+
+                if (storedPolicyName == null)
```

Comment:
@khoanguyen102 storedPolicyName is never used in our lib, we should remove this.
Or if you really want to keep this, add another overload to GetBlobUrlAsync that has this parameter instead of date too.

Reply:
I have updated it